docs/Changelog.md

```diff
@@ -4633,6 +4633,43 @@ opset_import {
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Shape-1"></a>**Shape-1**</a>
+
+  Takes a tensor as input and outputs an 1D int64 tensor containing the shape of the input tensor.
```

Comment:
Why 1D tensor I thought ONNX supports scalars as 0-dimensional tensors

Reply:
Updated to return a scalar

Reply:
@bddppq Right, I meant the `Size` op.